external/build/youtube-playlist.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * @fileoverview Fetches the most recent playlists by specified channels from YouTube
+ */
+
+require('dotenv').config();
+
+const fs = require('fs').promises;
+const path = require('path');
+const ms = require('ms');
+const {google} = require('googleapis');
+
+/**
+ * Time interval to determine if the current data set is stale
+ */
+const FETCH_INTERVAL = ms('4h');
+
+/**
+ * Config constants to interact with the YouTube API
+ */
+const API_KEY = process.env.YOUTUBE_API_KEY;
+const PART = ['snippet'];
+const MAX_RESULTS = 50;
+
+/**
+ * Set Google Chrome Developers Youtube channel ID (https://www.youtube.com/@ChromeDevs)
+ */
+const CHANNELS = ['UCnUYZLuoy1rq1aVMwx4aTzw'];
+
+/**
+ * Target directory for the fetched data
+ */
+const targetFile = path.join(__dirname, '../data/youtube-playlist.json');
+const currentTimestamp = Date.now();
+
+const result = {};
+
+const youtube = google.youtube('v3');
```

Comment:
I think I'd merge this with the require above, so `const youtube = require('googleapis').google.youtube('v3')` - what a weird way to structure a package anyway.